### DISCUSSION TITLE
Try to add admob lite version

### DISCUSSION THREAD

**Discussion by merbin2012** [EMI-INDO/emi-indo-cordova-plugin-admob#40] [Ideas]
The lite version reduces the app size by up to 2.5MB. Allowing users to choose their preferred version would make it even more useful.

I think we can add an additional variable to choose the version for Cordova while installing the plugin, and we can add one more property in `capacitor.config.json` then control it in the hook (`capacitor-hook-admob-ids.js`)
```
  "plugins": {
    "AdMob": {
      "APP_ID_ANDROID": "ca-app-pub-xxxx",
      "APP_ID_IOS": "ca-app-pub-xxxxx",
      "VERSION":"lite"
    },
```

`<framework src="com.google.android.gms:play-services-ads:$PLAY_SERVICES_VERSION" />`
and
`<framework src="com.google.android.gms:play-services-ads-lite:$PLAY_SERVICES_VERSION" />`

**Comment by EMI-INDO** [comment]
I'll give it a try

**Reply by 1n3JgKl9pQ6cUMrW** [reply]
Don't add too much features in the plug-in no-one uses.

I'd rather prefer a solid plug-in without too much options, which is maintained steady in the nearest future, than a plug-in loaded with options that becomes so difficult to maintain, integrate and update no-one is able to use it.

I really appreciate your hard work and I think your Abmob-plug-ins are very important for the community, but development should be easy and steady, no rush rush or hurry hurry.

Me myself has more than 40 years of professional experience in coding, development, maintaining projects and plug-ins like these (for various frameworks since the '80s). And it's very hard to develop one that's future proof and will still work over about 5 years... trust me.

**Reply by merbin2012** [reply]
So many people are unaware of this lite version, which is why it isn't widely used. Moreover, the lite version is better than the standard version. Saving 2.5MB in an app is not a small matter.

**Comment by EMI-INDO** [comment]
I will look for solutions, reduce, maintain, add any options that will not create confusion, that will benefit people.

I will create code injections from a separate npm Id to maintain all the current code.

**Reply by 1n3JgKl9pQ6cUMrW** [reply]
That's cool :-)

But it's hard work and lot's of work to maintain a complex plug-in like this. Especially for years.

I've been working with Cordova for about 15 years now and most plug-ins died, because lack of maintenance.

Of course we can fork yours, but at the end that won't guaranty long-term development for the community.

So keep up the good work, and don't make it too hard for yourself!

**Comment by EMI-INDO** [comment]
@merbin2012 

```
npm i capacitor-hook-admob-ids
```

https://www.npmjs.com/package/capacitor-hook-admob-ids

**Comment by EMI-INDO** [comment]
### sorry the documentation is not correct 
- which is correct like this

```
 "scripts": {
	.....
	.....
        "build": "npm run update-admob && cross-env NODE_ENV=production vite build",
       "update-admob": "node node_modules/capacitor-hook-admob-ids/hooks/capacitor-hook-admob-ids.js,
	.....
	.....
  },
```

**Reply by merbin2012** [reply]
I do not understand what you mean.

![packagejson](https://github.com/user-attachments/assets/b6938233-e9ed-4202-a83d-7983b83f9475)

**Comment by EMI-INDO** [comment]
### I mean if you want to use the lite option automatically.


```
npm i capacitor-hook-admob-ids
```


### Replace 
`"update-admob": "node node_modules/emi-indo-cordova-plugin-admob/capacitor-hook-admob-ids.js"`
### to
`"update-admob": "node node_modules/capacitor-hook-admob-ids/hooks/capacitor-hook-admob-ids.js"`

### documentation
https://www.npmjs.com/package/capacitor-hook-admob-ids

**Reply by merbin2012** [reply]
Ooh yes. Got it
